pkgs/development/python-modules/cryptography/default.nix

```diff
@@ -47,9 +51,11 @@ buildPythonPackage rec {
   nativeBuildInputs = lib.optionals (!isPyPy) [
     cffi
   ] ++ [
-    rustPlatform.cargoSetupHook
     setuptools-rust
-  ] ++ (with rustPlatform; [ rust.cargo rust.rustc ]);
+    rustPlatform.cargoSetupHook
+    rustc
+    cargo
```

Comment:
Can we assert/check in a setup hook this somewhere to make sure this is correct?

Reply:
Or should this be propagated inputs for `setuptools-rust`?